File: src/main/kotlin/com/nonnulldinu/clionmeson/errorHandler/TriggerErrorAction.java
package com.nonnulldinu.clionmeson.errorHandler;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import org.jetbrains.annotations.NotNull;

public class TriggerErrorAction extends AnAction {
	public TriggerErrorAction() {
		super("Error Reporting: Trigger Exception");
	}

	@Override
	public void actionPerformed(@NotNull AnActionEvent e) {
		throw new IllegalStateException("this is a bug!");
	}
}

File: src/main/kotlin/com/nonnulldinu/clionmeson/errorHandler/TriggerErrorAction.kt
package com.nonnulldinu.clionmeson.errorHandler

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent

class TriggerErrorAction : AnAction("Error Reporting: Trigger Exception") {
    override fun actionPerformed(e: AnActionEvent) {
        throw StackOverflowError("This is an artificial bug!")
    }
}